pypeit/reduce.py

```diff
@@ -190,64 +143,213 @@ def parse_manual_dict(self, manual_dict, neg=False):
         # Return
         return manual_extract_dict
 
-    def find_objects(self, image, std=False, ir_redux=False, std_trace=None, maskslits=None,
-                     show_peaks=False, show_fits=False, show_trace=False, show=False,
-                     manual_extract_dict=None, debug=False):
+    def extract(self, global_sky, sobjs_obj):
         """
+        Main method to extract spectra from the ScienceImage
 
         Args:
-            image:
-            std:
-            ir_redux:
-            std_trace:
-            maskslits:
-            show_peaks:
-            show_fits:
-            show_trace:
-            show:
-            manual_extract_dict:
-            debug:
+            global_sky (ndarray):
+                Sky estimate
+            sobjs_obj (SpecObjs):
+                List of SpecObj that have been found and traced
+        """
+        # This holds the objects, pre-extraction
+        self.sobjs_obj = sobjs_obj
+
+        # Boxcar only??
```

Comment:
Change this comment to Boxcar only with global sky subtraction.

Reply:
done.